roles/docker/tasks/package_docker.yml

```diff
@@ -3,6 +3,8 @@
   command: "{{ repoquery_cmd }} --installed --qf '%{version}' docker"
   when: not openshift.common.is_atomic | bool
   register: curr_docker_version
+  retries: 4
```

Comment:
Should this be modified to use the ``repoquery`` module with the updates in this PR?

Reply:
Probably. I think it's the only time this is used. I didn't want to take the time to figure out if there was any gotcha to doing that.